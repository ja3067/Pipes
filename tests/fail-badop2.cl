def dynamic():
	if x == 3:
		return 3
	else:
		return 4.0

x = 3
z = dynamic()

y = z + 3
print(y)

x = 4
z = dynamic()

y = z + 3
print(y)
