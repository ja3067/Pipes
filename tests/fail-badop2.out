6
RuntimeError: unsupported operand type(s) for binary +
